installer/helper.py

```diff
@@ -18,6 +19,13 @@
 except NameError:
     pass
 
+def install(package):
+    '''Install a package using pip or apt depending upon the platform'''
+    if platform == "linux" or platform == "linux2":
+        subprocess.check_call([sys.executable, "-m", "apt", "-q", "install", package])
```

Comment:
Hmm, apt is only for Debian/Ubuntu based distros. Won't work on Fedora, openSUSE, Arch, Gentoo, ...

Reply:
We can use **DISTRO** package to get distro id, and then use package manager acc. to distor id. (https://distro.readthedocs.io/en/latest/)

OR

We can add an statement for other users to use package manager based on their distros manually.

OR

We can make these options by default, and get user choice based on their distro.